django_event_sourcing/registers.py

```diff
@@ -17,6 +17,11 @@ def __init__(self, event_type_classes):
                 self.data[event_type.fully_qualified_value] = event_type
 
 
+RegisteredSideEffect = collections.namedtuple(
+    "RegisteredSideEffect", field_names=("callable", "condition")
+)
```

Comment:
It might be nicer to use [`NamedTuple`](https://docs.python.org/3/library/typing.html#typing.NamedTuple) here.